hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -311,7 +311,7 @@
         // The following three values get exposed in the life cycle CSRs
         // that are also readable via the TAP.
         SiliconCreatorId: "16'h 4001", // Creator ID for Earlgrey discrete integration
-        ProductId: "16'h 0001", // Earlgrey engineering sample
+        ProductId: "16'h 0002", // Earlgrey engineering sample
```

Comment:
nit: the comment should be updated too, right? IIUC the new number corresponds to the production silicon.

Reply:
Indeed, will do.